docs/DeveloperGuide.md

```diff
@@ -882,13 +927,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
```

Comment:
I feel the best way to deal with this is to have `1a. User has no contact to list` followed by `1a1. NetworkBook shows an empty list of contacts`.